pom.xml

```diff
@@ -92,9 +92,9 @@
         <accumulo.version>1.8.1</accumulo.version>
         <zookeeper.version>3.4.10</zookeeper.version>
         <thrift.version>0.9.3</thrift.version>
-        <hadoop.version>2.6.4</hadoop.version>
+        <hadoop.version>2.7.3</hadoop.version>
```

Comment:
if we need to upgrade hadoop versions, I'd say we go to 2.10.0 (the latest 2.x version as of right now). I know that there are some s3-related changes in 2.8 in particular that would be nice to have.